rotkehlchen/db/snapshots.py

```diff
@@ -297,13 +313,33 @@ def _import_snapshot(
                     usd_value=str(FVal(entry['usd_value'])),
```

Comment:
Why do this and not just pass the `entry['usd_value']`? is a string already right?

Reply:
Also can raise deserialization error but this would be catched by the schema and raise validationerror so I believe is not needed. But anyway the conversion is not needed since it happens in the schema validation or am I wrong?

Reply:
This method is also used when importing snapshots as CSV.

Reply:
Okay